src/main/java/uk/gov/hmcts/ccd/auditlog/AuditService.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.ccd.auditlog;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.auditlog.aop.AuditContext;
+import uk.gov.hmcts.ccd.data.SecurityUtils;
+import uk.gov.hmcts.ccd.data.user.CachedUserRepository;
+import uk.gov.hmcts.ccd.data.user.UserRepository;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+
+import static java.time.format.DateTimeFormatter.ISO_LOCAL_DATE_TIME;
+
+@Service
+public class AuditService {
+
+    private final Clock clock;
+    private final SecurityUtils securityUtils;
+    private final UserRepository userRepository;
+    private final AuditRepository auditRepository;
+
+    public AuditService(@Qualifier("utcClock") final Clock clock,
+                        @Qualifier(CachedUserRepository.QUALIFIER) final UserRepository userRepository,
+                        @Lazy final SecurityUtils securityUtils, final AuditRepository auditRepository) {
```

Comment:
we should use dependency injection for AuditRepository

Reply:
it does